tests/Spec.hs

```diff
@@ -234,7 +234,27 @@ main = hspec $ do
         actions0 = snd $ handleEventFlat event0 state
         actions1 = snd $ handleEventFlat event1 state
       actions0 `shouldBe` []
-      actions1 `shouldBe` [AIsReviewer "deckard"]
+      actions1 `shouldBe`
+        [ AIsReviewer "deckard"
+        , ALeaveComment (PullRequestId 1) "This PR was approved by @deckard.\n\nMerge queue is empty. Rebasing immediately."
+        ]
+
+    it "notifies approvers about queue position" $ do
+      let
+        state0 = candidateState (PullRequestId 1) (Branch "p") (Sha "a38") "tyrell" (Sha "84c")
+        state1 = Project.insertPullRequest (PullRequestId 2) (Branch "s") (Sha "dec") "Some PR" (Username "rachel") state0
```

Comment:
(It’s [Rach**a**el](https://en.wikipedia.org/wiki/List_of_Blade_Runner_characters#Rachael), not Rachel.)

Reply:
I was wondering what this was referring to!